roles/uvcmetrics/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+    uvcdat_src_path: "/opt/uvcdat-src"
+    uvcdat_build_path: "/opt/uvcdat-build"
+    uvcdat_install_path: "/opt/uvcdat"
```

Comment:
For the sake of consistency, let's default to installing under
```
/opt/uvcdat/{{ uvcdat_version }}
```
where `uvcdat_version` is also configurable (maybe default to "master").

Have a look at the [zookeeper-variables](https://github.com/opadron/gobig/blob/spark-role/roles/zookeeper-variables/tasks/main.yml) role (under my spark branch) for an example of what I've been doing with variables/directories/various options.